doc/reference_guide/source/index.rst

```diff
@@ -0,0 +1,39 @@
+.. fparser Reference Guide documentation master file.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+fparser Reference Guide
+########################
+
+Welcome to the fparser Reference Guide. This consists of auto-generated
+API documentation produced using both the :ref:`AutoAPI Sphinx plugin <autoapi>`
+and :ref:`Doxygen <doxygen>`.
+
+.. _autoapi:
+
+Sphinx and AutoAPI-generated Documentation
```

Comment:
Btw, many of the fparser classes are dynamically generated? I guess those won't appear here? If that's the case, can you add a note/warning explaining this